### PR TITLE
Convert HTML to PDF with DocRaptor

### LINKED ISSUE #4179: [Languages] Convert HTML to PDF with DocRaptor
#### Proposed title of article
Convert HTML to PDF with DocRaptor

#### Proposed article introduction
Producing PDFs is a typical programming task. PDFs are utilized to make solicitations, receipts, eBooks, distributing reports and the sky is the limit from there. Tragically, automatic age of PDF can be troublesome, particularly in the event that you need to create numerous PDFs quickly. 
DocRaptor is a HTML-to-PDF API that extraordinarily improves on the issue, and they have a PHP library for simple joining. In this instructional exercise, learners will be able to  stroll through how to change over HTML into a PDF with DocRaptor:
#### Key takeaways 
After reading this article learners will be conversant in:
- Installation
- Authentication
- Add HTML or a URL
- API Options
- Get the document!

#### Article quality
This article will help learners with the basics they need to get started with DocRaptor. Many articles out there  only rely on official documentation and some of the examples they offer are a bit too harsh for novices in PHP. This article will be ideal for novices and the code snippets involved will be illustrated clearly and will be straight on point.

#### References
N/A

### PR DESCRIPTION
**Pull request by taves-hub**
This fixes #4179

### CODE REVIEW

**Comment by taves-hub** [comment]
@hectorkambow, @mercymeave, I have done the required changes on my article. You may have a look at it when you are free

**Comment by mercymeave** [comment]
Hello @taves-hub

**Comment by mercymeave** [comment]
I will check it out.

**Review by mercymeave — APPROVED** [review]
Take note of the changes made for clearer articles in the future.

**Comment by kymomolly** [comment]
@mercymeave, have taken note, for a better article in the future. Thanks

**Comment by taves-hub** [comment]
@mercymeave, have taken note, for a better article in the future. Thanks

**Comment by taves-hub** [comment]
@mercymeave, I think you forgot to include tech tutorial label

**Comment by taves-hub** [comment]
@mercymeave, I think you forgot to include tech tutorial label

**Comment by taves-hub** [comment]
@mercymeave, I think you forgot to include tech tutorial label

**Comment by taves-hub** [comment]
@mercymeave, hello was inquiring about the `tech tutorial` label. Thank you

**Comment by lalith1403** [comment]
jumping on this one 👍

File: content/articles/convert-html-to-pdf-with-docraptor/index.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+- [Table of content](#table-of-content)
+- [Installation](#installation)
+- [API Authentication](#api-authentication)
+- [Adding HTML content or a URL](#adding-html-content-or-a-url)
+- [API document options](#api-document-options)
+- [Get the API documentation](#get-documentation-for-the-library)
+- [Conclusion](#conclusion)
+
+### Installation
+To get started with Docraptor, we need to install it in our local machine. Installation can be done in two ways. The first way is using composer dependency manager, while the second way is by downloading a zip file of the library and extracting it in the specific project folder.
```
[the remark this answers is not in the document]
Comment (lalith1403) [reply content/articles/convert-html-to-pdf-with-docraptor/index.md:37]: give a bried intro to compose dependency manager.

File: content/articles/convert-html-to-pdf-with-docraptor/index.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+#### Downloading using composer
+- Download composer from this [link](https://getcomposer.org/download/) then execute the command below to install Docraptor.
+
+```php
+composer require docraptor/docraptor
+```
+
+#### Installation using  a zip file
+Navigate through [this](https://docraptor.com/documentation) link to find the installation zip file. Download the file, then extract it into your workspace. Create a new file and name it `docraptor.php`, and include "autoload.php" in it
```
Comment (lalith1403) [inline content/articles/convert-html-to-pdf-with-docraptor/index.md:47]: can you provide the link to the zip file?

Comment (lalith1403) [reply content/articles/convert-html-to-pdf-with-docraptor/index.md:47]: `autoload.php`

File: content/articles/convert-html-to-pdf-with-docraptor/index.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+```php
+//docraptor.php
+require_once('./path/to/docraptor-php/autoload.php');
+```
+
+### API authentication
+Every project where an API is used needs an API key. The key identifies the machine from where the request for specific resources is coming. Our project is not an exception; therefore, we need to add an API Key to authenticate the source of our help. We will use the phrase "KEY YOUR API HERE" as our API key, which we get after sending an HTTP request. This key, however, only allows us to use watermarked documents.
```
Comment (lalith1403) [inline content/articles/convert-html-to-pdf-with-docraptor/index.md:55]: API key. Avoid unnecessary capitalization.

Comment (lalith1403) [reply content/articles/convert-html-to-pdf-with-docraptor/index.md:55]: What do you mean by you get after sending an HTTP request? It is not clear.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Please rectify the errors that exist in the article. There are several readbility issues. You may rectify them and create a new PR. I will be closing this one. Once you create a new PR, we will review the article one more time to make sure we give the author another opportunity to make changes. We encourage you to be diligent about rectifying all the errors and only publishing a polished article. Thank you.

@mercymeave There are many errors in the article. There were spelling mistakes in the article. Please look into the changes made.

**Comment by hectorkambow** [comment]
>NOTE: A duplicate PR will go back into the queue and will not receive any further major revisions. A duplicate PR should be 100% ready to be published, otherwise it will be closed immediately. We encourage our new contributors to go over our [resources page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) and [new contributor folder](https://github.com/section-engineering-education/engineering-education/tree/master/new_contributors) to be better familiar with the level of quality and style we are looking to publish.
 
Closing a PR does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.
 
👍

**Comment by taves-hub** [comment]
@lalith1403, How are you doing sir? I was just requesting if you may open this PR so that I may push the requested changes that you have suggested. I have already corrected the changes and any other that may have appeared. I would really appreciate it. thank you

**Comment by lalith1403** [comment]
As per the guidelines of the community, we request you to open a new PR and ping me. Also, note the comment by @hectorkambow. Thanks.

> > NOTE: A duplicate PR will go back into the queue and will not receive any further major revisions. A duplicate PR should be 100% ready to be published, otherwise it will be closed immediately. We encourage our new contributors to go over our [resources page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) and [new contributor folder](https://github.com/section-engineering-education/engineering-education/tree/master/new_contributors) to be better familiar with the level of quality and style we are looking to publish.
> 
> Closing a PR does NOT mean the article can not be published (or fixed) it simply means that the edits, revisions, and fixes will happen outside of the queue.
> 
> 👍

**Comment by taves-hub** [comment]
@lalith1403, Okay.

**Pull request closed without merging**